src/authz-module/components/ResourceTooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import { Icon, OverlayTrigger, Popover } from '@openedx/paragon';
+import { Info } from '@openedx/paragon/icons';
+import { PermissionsResourceGrouped, RoleResourceGroup } from '@src/types';
+
+type ResourceTooltipProps = {
+  resourceGroup: PermissionsResourceGrouped | RoleResourceGroup;
+};
+
+const ResourceTooltip = ({ resourceGroup }:ResourceTooltipProps) => (
+  <OverlayTrigger
+    key={`overlay-${resourceGroup.key}`}
+    placement="auto"
+    overlay={(
+      <Popover variant="light" id={`tooltip-${resourceGroup.label}`}>
```

Comment:
```suggestion
      <Popover id={`tooltip-${resourceGroup.label}`}>
```

`light` isn't a valid `Popover` variant. The variants are `success`, `warning`, `danger`, and no variant (which is what this is rendering as now).

https://paragon-openedx.netlify.app/components/popover/#state-variants

https://github.com/openedx/paragon/blob/cce7beb990c61778616ae7d7216ff4fbc609a100/src/Popover/index.jsx#L23

https://github.com/openedx/paragon/blob/cce7beb990c61778616ae7d7216ff4fbc609a100/src/Popover/index.scss#L20-L30